moda-contracts/test/SplitsFactoryWithFork.t.sol

```diff
@@ -40,13 +39,14 @@ contract SplitsFactoryWithForkTest is Test {
     Registry _registry;
     ISplitMain _splitMain;
 
-    string EVM_RPC_URL = vm.envString("EVM_RPC_URL");
+    string EVM_RPC_URL = vm.envString("MUMBAI_RPC_URL");
```

Comment:
I think we should change it back to the original name. Let's remove the MUMBAI... vars from the env files too